objects/_switch.scss

```diff
@@ -14,21 +19,75 @@ $switch-knob-size: $switch-height - ($switch-inner-padding * 2);
     width: $switch-width;
     background: $grey-light;
     cursor: pointer;
+
     &.is-on {
         background: $blue;
     }
 }
-    .switch__knob {
-        @include border-radius($switch-height / 2);
-        @include single-transition(0.1s linear left);
+
+.switch__knob {
+    @include border-radius(50%);
+    @include single-transition(0.1s linear left);
+    position: absolute;
+    display: block;
+    left: $switch-inner-padding;
+    top: $switch-inner-padding;
+    height: $switch-knob-size;
+    width: $switch-knob-size;
+    background-color: $white;
+
+    .is-on & {
+        left: $switch-width - $switch-knob-size - $switch-inner-padding;
+    }
+}
+
+.switch--checkmark {
+    width: $switch-width-with-icon;
+
+    &::before {
+        font-family: 'ddg-serp-icons';
+        speak: none;
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
+        content: "\2713";
```

Comment:
is it possible to use the icon include? e.g. `icon("check")`  Or does that not work for some reason?

Reply:
I didn't know about icon("check")! Thanks :)